pithos/pandora/pandora.py

```diff
@@ -420,9 +420,9 @@ def title(self):
                 self._title = self.songName
             else:
                 try:
-                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data:
-                        dom = minidom.parseString(xml_data.read())
-                    attr_value = dom.getElementsByTagName('songExplorer')[0].attributes['songTitle'].value
+                    with urllib.request.urlopen(self.songExplorerUrl) as xml_data, \
+                    minidom.parseString(xml_data.read()) as dom:
```

Comment:
Ok @TingPing your auto PEP8 checker can't decide? If I indent this it's over intended of I don't it's missing the indentation?

To me it looks better intended but whatever works...